tests/Data/InputsTest.elm

```diff
@@ -40,12 +40,13 @@ suite =
                         ]
                     ]
                 , describe "Query countries validation"
-                    [ { tShirtCotonAsie | countries = List.map Country.Code [ "FR", "CN", "CN", "CN", "FR", "FR" ] }
+                    [ { tShirtCotonAsie | countryFabric = Country.Code "CN", countryDyeing = Country.Code "CN", countryMaking = Country.Code "CN" }
```

Comment:
These are rather long lines. Options :

- Spread them over multiple lines through elm-format
- Create a short helper to build `Country.Code`

Your call!

Reply:
I went with first option :+1: